nixos/modules/system/boot/loader/generations-dir/generations-dir.nix

```diff
@@ -47,6 +47,28 @@ in
         '';
       };
 
+      copyDefault = mkOption {
+        default = versionOlder config.system.stateVersion "22.11";
+        defaultText = literalMD ''
+          `false` (or `true` for `system.stateVersion` older than 22.11)
+        '';
```

Comment:
```suggestion
        defaultText = literalMD ''
          versionOlder config.system.stateVersion "22.11"
        '';
```

I still don't like this but the text before was confusing and it was not clear how the default works.

Reply:
Should be `literalExpression` then

Reply:
Switched this to the literal expression.